project/Dependencies.scala

```diff
@@ -24,8 +24,7 @@ object Dependencies {
 
   val akkaDependencies: Seq[ModuleID] = Seq(
     "com.typesafe.akka" %% "akka-actor" % versions.akka,
-    "com.typesafe.akka" %% "akka-agent" % versions.akka,
-    "com.typesafe.akka" %% "akka-stream" % versions.akka
+    "com.typesafe.akka" %% "akka-agent" % versions.akka
```

Comment:
Why take out the akka streams dependency entirely?

Reply:
That’s almost certainly a rebase bug.